polkadot/xcm/pallet-xcm/src/lib.rs

```diff
@@ -199,6 +209,9 @@ pub mod pallet {
 		/// An implementation of `Get<u32>` which just returns the latest XCM version which we can
 		/// support.
 		pub const CurrentXcmVersion: u32 = XCM_VERSION;
+
+		/// The maximum number of distinct locations allowed as authorized aliases for a local origin.
+		pub const MaxAuthorizedAliases: u32 = 10;
```

Comment:
What if I have an account on AH and want to setup aliases for all my accounts on other sibling parachains? Why only 10? Is 10 enough?

Reply:
I want to be pragmatic here. I believe 10 remote aliases is more than enough in practice. I expect users of this feature to have only a couple of authorized aliasers.

If we see anyone needing more, we can simply bump it up. In the meantime, let's keep it low (so that tx is also cheap).